cmd/admin-policy-list.go

```diff
@@ -76,11 +77,15 @@ func mainAdminPolicyList(ctx *cli.Context) error {
 	fatalIf(probe.NewError(e).Trace(args...), "Cannot list policy")
 
 	if policyName := args.Get(1); policyName != "" {
-		printMsg(userPolicyMessage{
-			op:         "list",
-			Policy:     policyName,
-			PolicyJSON: policies[policyName],
-		})
+		if len(policies[policyName]) != 0 {
+			printMsg(userPolicyMessage{
+				op:         "list",
+				Policy:     policyName,
+				PolicyJSON: policies[policyName],
+			})
+		} else {
+			fatalIf(probe.NewError(fmt.Errorf("The specified policy: %s not found", policyName)), "Cannot find policy")
```

Comment:
Neat change: The error message can be better this way:

`fatalIf(probe.NewError(fmt.Errorf("`%s` is not found", policyName)), "Cannot list the policy")`

Reply:
Done.